src/app/lib/config/services/arabic.js

```diff
@@ -203,7 +203,8 @@ export const service = {
     footer: {
       trustProjectLink: {
         href: 'https://www.bbc.com/arabic/institutional-49283069',
-        text: 'لماذا يمكنك الإعتماد على أخبار بي بي سي',
+        text:
+          'لماذا يمكنك الإعتماد على أخبار بي بي سيلماذا يمكنك الاعتماد على أخبار بي بي سي',
```

Comment:
```suggestion
          'لماذا يمكنك الإعتماد على أخبار بي بي سي',
```
This should fix it - it was pasted exactly twice.

Reply:
Just done locally and updated snapshot, pushing now!